### ISSUE TITLE
Change layer for intermittent natural=water

### ISSUE THREAD

**Issue by maro-21** [openstreetmap/iD#10186]
Features tagged `natural=water` + `intermittent=yes` are in the Past/Future layer now. I think the better place would be the Water layer.

**Comment by tyrasd** [comment]
yeah, it's probably best to have these features only in the "water" feature type
